plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
```

Comment:
The documentation is plug file is external documentation, it is not used in the help command. The documentation that is used in help command is the docstrings, they can be fairly independent and fwiw we don't have to maintain documentation of `plug` file